src/protocol.vala

```diff
@@ -1,3 +1,23 @@
+/* protocol.vala
+ *
+ * Copyright 2017-2019 Ben Iofel <iofelben@gmail.com>
```

Comment:
Oh actually, seems like your mail address changed, @benwaffle . Do you prefer this email or `ben@iofel.me`?

Reply:
ben@iofel.me